dist/tools/pic32prog/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME     = pic32prog
+PKG_URL      = https://github.com/sergev/pic32prog
+PKG_VERSION  = f5d27d6c9506bc0d1f2053002e2aadad1ac424cd
```

Comment:
I tried current upstream, also works. current master commit is "b9f8db3b352804392b02b42475fc42874ac8bf04"

Reply:
Indeed, the new version looks like it handles new boards and compilation issues.